linked_list/main.py

```diff
@@ -62,34 +68,166 @@ class LinkedList(AbstractLinkedList):
     """
 
     def __init__(self, elements=None):
-        pass
+        self.elements = []      
+        if (elements == None or len(elements) <1):
+            #self.start, self.end = Node(None, None), Node(None, None)
+            self.start, self.end = None, None
+        if elements != None and len(elements) > 0:
+            self.elements = elements
+            index = 1
+            for e in range(len(self.elements)):
+                try:
+                    e = Node(e, Node(self.elements[index]))
```

Comment:
It looks like the idea was to use index to keep track of what the next Node would be, but index would need to be incremented in the for loop to stay 1 ahead of e